addons/modules/XEH_preInit.sqf

```diff
@@ -6,4 +6,15 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+["ModuleCurator_F", "Init", {
+    params ["_logic"];
+
+    _logic addEventHandler ["CuratorObjectPlaced",
+    {
+        if (param[1] isKindOf "Module_F") then {
+            _this call FUNC(handleModulePlaced);
```

Comment:
Don't think we need a separate function for this. Can just do the `&& {!(_logic isKindOf QGVAR(moduleBase))}` check in the if statement above.